stdlib/public/Cxx/std/String.swift

```diff
@@ -161,6 +161,26 @@ extension std.u32string: Equatable {
   }
 }
 
+// MARK: Comparable conformance for C++ strings
+
+extension std.string: Comparable  {
```

Comment:
Minor request: could you please merge this extension with the extension above that adds the `Equatable` conformance? `Equatable` and `Comparable` are closely related, so I think we can have both conformances in one extension.

(same for the two extensions below)

Reply:
That makes sense, sure!